ui-v2/app/styles/routes/dc/services/index.scss

```diff
@@ -1,3 +1,15 @@
+// Services - Linked Services tab
+.gateway-services-list > ul {
+  @extend %gateway-services-list;
+}
+%gateway-services-list > li:not(:first-child) {
+  @extend %gateway-service-row;
+}
+%gateway-service-row {
+  @extend %composite-row, %with-composite-row-intent;
+}
```

Comment:
I think we don't have to make new `%placholders` for all of these we could just do:
```
.gateway-services-list,
.consul-services-list,
.consul-instances-list,
.etc {
  @extend %composite-row, %with-composite-row-intent;
}
```

..all done inside a `./styles/components/composite-row.scss`.

I suppose my thoughts are, theres a lot of boilerplate for each one of these individual things, when they are all essentially the same thing. We can chat about that another time though.

Reply:
Yes, I agree. We can refactor in the next composite-row implementation.